http/swagger.yml

```diff
@@ -6551,6 +6551,9 @@ components:
         id:
           readOnly: true
           type: string
+        type:
+          description: The type of task, this can be used for filtering
+          type: string
```

Comment:
Is there a fixed set of values that this `type` can be? If so, it would be helpful to add an `enum` field here.

Reply:
Anything except the wildcard "*" no limits currently.